Source/Core/Core/HW/SystemTimers.cpp

```diff
@@ -79,23 +77,23 @@ static int et_IPC_HLE;
 static int et_PatchEngine; // PatchEngine updates every 1/60th of a second by default
 static int et_Throttle;
 
+static u32 s_cpu_core_clock = 486000000u;             // 486 mhz (its not 485, stop bugging me!)
 static u64 s_last_sync_gpu_tick;
 
-// These are badly educated guesses
-// Feel free to experiment. Set these in Init below.
-static int
-	// This is a fixed value, don't change it
-	AUDIO_DMA_PERIOD,
+// These two are badly educated guesses.
+// Feel free to experiment. Set them in Init below.
 
-	// This is completely arbitrary. If we find that we need lower latency, we can just
-	// increase this number.
-	IPC_HLE_PERIOD;
+// This is a fixed value, don't change it
+static int s_audio_dma_period;
+// This is completely arbitrary. If we find that we need lower latency,
+// we can just increase this number.
+static int s_ipc_hle_period;
```

Comment:
Hm, this one is only initialized (and used, hopefully) when ```SConfig::bWii``` is true. Not sure if thats a good or bad thing, but I can't think of a different way to doing this either atm.